.azure-pipelines-templates/cmake.yml

```diff
@@ -14,9 +14,3 @@ steps:
   env:
```

Comment:
Should the `cat /proc/cpuinfo` step move to the new `ninja.yml`?

Reply:
Should it? I'm not sure why we had it in the first place, I guess to debug differences between machines, but if anything it seems like it should be in the test phase, when we actually run stuff?

Reply:
We had it to check that the CI machines supported a specific extension for RNG. It may not be required anymore?